### PR TITLE
Adding frontend for Pyramid 

### PR DESCRIPTION
**Pull request by MayKhine**

### CODE REVIEW

File: packages/frontend/src/components/InputArr.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+
+export const InputArr = () => {
+  const [dataArr, setDataArr] = useState<Array<Array<string>>>([
+    ["a", "", "", "c", ""],
+    ["", "", "", ""],
+    ["", "", ""],
+  ]);
```
Comment (MayKhine) [inline packages/frontend/src/components/InputArr.tsx:8]: remove

File: packages/frontend/src/components/InputBox.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export const InputBox = () => {
```
Comment (MayKhine) [inline packages/frontend/src/components/InputBox.tsx:1]: remove

File: packages/frontend/src/components/Pyramid.tsx [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+import { useEffect, useRef, useState } from "react";
+import { InputArr } from "./InputArr";
+import { Button } from "./Button";
+import { tokens } from "../tokens.stylex";
+import { PopUp } from "./PopUp";
+
+type PyramidType = {
+  data: PyramidPrompt;
+};
+export const Pyramid = ({ data }: PyramidType) => {
```
Comment (MayKhine) [inline packages/frontend/src/components/Pyramid.tsx:15]: remove the data, pull data from backend

File: packages/frontend/src/components/Pyramid_old.tsx [hunk 4]
```diff
@@ -0,0 +1,251 @@
+import * as stylex from "@stylexjs/stylex";
```
Comment (MayKhine) [inline packages/frontend/src/components/Pyramid_old.tsx:1]: remove, old file.

**Review by EthanSherr — APPROVED** [review]
lgtm

**Pull request merged**